### PR TITLE
Updating Orc8r deployment how-to

### PR DESCRIPTION
**Pull request by Gmerold**
Updates Orc8r deployment how-to with info about Fluentd integration.
Adds new doc describing Orc8r - ES integration.

### CODE REVIEW

File: docs/how-to/deploy_charmed_magma_orchestrator.md [hunk 1]
```diff
@@ -18,9 +18,18 @@ Create an `overlay.yaml` file that contains the following content:
 
 ```yaml title="overlay.yaml"
 applications:
+  fluentd:
+    options:
+    domain: <your domain name>
+    elasticsearch-url: <your elasticsearch https url>
+    fluentd-chunk-limit-size: "2M"
```
Comment (gruyaume) [inline docs/how-to/deploy_charmed_magma_orchestrator.md]: Do we need `fluentd-chunk-limit-size` and `fluentd-queue-limit-length` here?

File: docs/how-to/deploy_charmed_magma_orchestrator.md [hunk 2]
```diff
@@ -34,6 +43,11 @@ applications:
 
     This configuration is unsecure because it uses self-signed certificates.
 
+!!! info
+    
+    Elasticsearch is not part of the magma-orc8r bundle and needs to be deployed prior to deploying 
+    the bundle. Elasticsearch needs to support both `http` and `https` requests.
```
Comment (gruyaume) [inline docs/how-to/deploy_charmed_magma_orchestrator.md]: Here we should point to a document "how-to: integrate charmed orchestrator with Elasticsearch"...and we should create such a document.

File: docs/how-to/deploy_charmed_magma_orchestrator.md (hunk 1 above)
Comment (Gmerold) [reply docs/how-to/deploy_charmed_magma_orchestrator.md]: Not necessarily. I put it here to let the user know that such options are available.

File: docs/how-to/deploy_charmed_magma_orchestrator.md (hunk 2 above)
Comment (Gmerold) [reply docs/how-to/deploy_charmed_magma_orchestrator.md]: I thought about it too, but...
1. It's impossible to write a universal guide to setting up an Elasticsearch. Should the document favor some option (i. e. Amazon's OpenSearch Service)?
2. Integration between charmed magma and the ES is done only through the overlay, which is already shown in this document. In this case, our how-to would practically become a guide to deploying the ES.

File: docs/how-to/deploy_charmed_magma_orchestrator.md (hunk 1 above)
Comment (gruyaume) [reply docs/how-to/deploy_charmed_magma_orchestrator.md]: I'd keep the content here to a minimum

File: docs/how-to/deploy_charmed_magma_orchestrator.md (hunk 2 above)
Comment (gruyaume) [reply docs/how-to/deploy_charmed_magma_orchestrator.md]: What if elasticsearch is deployed at a later moment? That should work as well right? Then this overlay wouldn't be of too much help

File: docs/how-to/deploy_charmed_magma_orchestrator.md (hunk 1 above)
Comment (Gmerold) [reply docs/how-to/deploy_charmed_magma_orchestrator.md]: Removed.

File: docs/how-to/deploy_charmed_magma_orchestrator.md (hunk 2 above)
Comment (Gmerold) [reply docs/how-to/deploy_charmed_magma_orchestrator.md]: Makes sense. New doc added.

File: docs/how-to/deploy_charmed_magma_orchestrator.md [hunk 3]
```diff
@@ -34,6 +41,11 @@ applications:
 
     This configuration is unsecure because it uses self-signed certificates.
 
+!!! info
+    
+    ElasticSearch is not part of the magma-orc8r bundle and needs to be deployed separately.
```
Comment (gruyaume) [inline docs/how-to/deploy_charmed_magma_orchestrator.md]:
```suggestion
    Elasticsearch is not part of the magma-orc8r bundle and needs to be deployed separately. 
```

File: docs/how-to/deploy_charmed_magma_orchestrator.md [hunk 4]
```diff
@@ -34,6 +41,11 @@ applications:
 
     This configuration is unsecure because it uses self-signed certificates.
 
+!!! info
+    
+    ElasticSearch is not part of the magma-orc8r bundle and needs to be deployed separately. 
+    For details regarding ElasticSearch integration please visit [Integrate Charmed Magma Orchestrator to ElasticSearch](integrate_charmed_magma_orchestrator_to_elasticsearch.md)
```
Comment (gruyaume) [inline docs/how-to/deploy_charmed_magma_orchestrator.md]:
```suggestion
    For details regarding Elasticsearch integration please visit [Integrate Charmed Magma Orchestrator to Elasticsearch](integrate_charmed_magma_orchestrator_to_elasticsearch.md)
```

File: mkdocs.yml [hunk 5]
```diff
@@ -9,6 +9,7 @@ nav:
         - "Deploy Charmed Magma Access Gateway": 'how-to/deploy_charmed_magma_access_gateway.md'
         - "Integrate Charmed Magma Access Gateway to Orchestrator": 'how-to/integrate_charmed_magma_access_gateway_to_orchestrator.md'
         - "Integrate Charmed Magma Access Gateway to srsRAN": 'how-to/integrate_charmed_magma_access_gateway_to_srsRAN.md'
+        - "Integrate Charmed Magma Orchestrator to ElasticSearch": 'how-to/integrate_charmed_magma_orchestrator_to_elasticsearch.md'
```
Comment (gruyaume) [inline mkdocs.yml]:
```suggestion
        - "Integrate Charmed Magma Orchestrator to Elasticsearch": 'how-to/integrate_charmed_magma_orchestrator_to_elasticsearch.md'
```

File: docs/how-to/integrate_charmed_magma_orchestrator_to_elasticsearch.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+# Integrate Charmed Magma Orchestrator to ElasticSearch
+
+## Requirements
+
+ - ElasticSearch instance (**not part of the `magma-orc8r` bundle**) supporting both `HTTP`
```
Comment (gruyaume) [inline docs/how-to/integrate_charmed_magma_orchestrator_to_elasticsearch.md]:
```suggestion
 - Elasticsearch instance supporting both `HTTP` 
 - Charmed Orchestrator
```

File: docs/how-to/deploy_charmed_magma_orchestrator.md (hunk 3 above)
Comment (Gmerold) [reply docs/how-to/deploy_charmed_magma_orchestrator.md]: Done.

File: docs/how-to/deploy_charmed_magma_orchestrator.md (hunk 4 above)
Comment (Gmerold) [reply docs/how-to/deploy_charmed_magma_orchestrator.md]: Done.

File: mkdocs.yml (hunk 5 above)
Comment (Gmerold) [reply mkdocs.yml]: Done.

File: docs/how-to/integrate_charmed_magma_orchestrator_to_elasticsearch.md (hunk 6 above)
Comment (Gmerold) [reply docs/how-to/integrate_charmed_magma_orchestrator_to_elasticsearch.md]: Done.

**Review by gruyaume — APPROVED** [review]

**Pull request merged**